modules/70_water/heat/realization.gms

```diff
@@ -6,10 +6,24 @@
 *** |  Contact: remind@pik-potsdam.de
 *** SOF ./modules/70_water/heat/realization.gms
 
-*' @description Exogenous water demand is calculated based on data on water demand coefficients and cooling shares. 
-*' Vintage structure in combination with time dependent cooling shares as vintages and efficiency factors are also considered.
-*' Demand is a function of excess heat as opposed to electricity output.
-*' @limitations Water demand is calculated in a post-processing of REMIND and not part of the optimization.
+*' @description Water demand is calculated based on data on water demand coefficients (both for consumption 
+*' and withdrawal) and cooling shares.
+*' 
+*' For thermal power plants, water demand is proportional to excess heat production, which is calculated as fuel input minus
+*' electricity output and smokestack losses (assumed to be 10% of fuel input for non-nuclear thermal power plants).
+*'
+*' For non-biomass renewable technologies, water demand is proportional to electricty output.
```

Comment:
is it subdivided further here? solar pv should in theory have less water demand than non-thorium nuclear...